include/lsst/afw/table/io/FitsSchemaInputMapper.h

```diff
@@ -64,6 +92,19 @@ class FitsSchemaInputMapper {
 public:
     typedef FitsSchemaItem Item;
 
+    /**
+     *  When processing each column, ask CFITSIO to read this many rows of
+     *  values from that column in a single call.
+     *
+     *  Both FITS binary tables and afw.table are stored row-major, so setting
+     *  this to something other than 1 leads to nonsequential reads.  But
+     *  given the way the I/O code is structured, we tend to get nonsequential
+     *  reads anyway, and it seems the per-call overload to CFITSIO is
+     *  sufficiently high that it's best to have *significantly*
+     *  non-sequential reads by making this >> 1.
```

Comment:
Add reasons not to set it to infinity?

Reply:
@erykoff's tests suggest that making it too large makes it go slower, especially when the width of the record is also large.  I'd guess that has something to do with disk page or CFITSIO buffer sizes.